release/goreleaser.opm.Dockerfile

```
@@ -2,7 +2,11 @@
 #   build opm images. See the configurations in .goreleaser.yaml
 #   and .github/workflows/release.yaml.
 
-FROM --platform=$BUILDPLATFORM ghcr.io/grpc-ecosystem/grpc-health-probe:v0.4.4 as grpc_health_probe
+FROM ubuntu as grpc_health_probe
+RUN apt update && \
+  apt install -y curl &&
+  curl -o /grpc_health_probe \
+  https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v0.4.5/grpc_health_probe-linux-$TARGETARCH
```

Does the following do the right thing?
```suggestion
FROM --platform=$TARGETPLATFORM ghcr.io/grpc-ecosystem/grpc-health-probe:v0.4.4 as grpc_health_probe

---

yep but grpc-health-probe isn't multi-arch it is build only for linux/amd64 https://github.com/grpc-ecosystem/grpc-health-probe/blob/master/.github/workflows/release.yml#L29

---

You're right. I swear I saw something about it being multi-arch when I was originally putting this together, but apparently not.

Disregard my suggestion!